styles.css

```diff
@@ -1,77 +1,98 @@
+body {
+  background-color: #172b4d;
+}
+
 .container {
-    border: 5px solid #344563;
-    width: 300px;
-    height: 100vh;
-    margin: 0 auto;
-    padding: 20px;
-    background-image: url(/Graphics/header-shapes\ mobile.png);
-    background-size: contain;
+  margin: 0 auto;
+  width: 315px;
+  height: 710px;
+  background-color: #fff;
+  background-image: url(./Graphics/header-shapes\ mobile.png); /* fixed the path */
+  background-size: contain;
+  background-position: center;
+  padding: 30px;
+  display: flex;
+  flex-direction: column;
+}
 
+.status {
+  align-self: flex-start;
 }
 
 .elements {
-    display: flex;
-    justify-content: space-between;
-    align-items: flex-start;
-
+  margin-top: -20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
 }
 
 .item {
-    order: 1;
-    margin-top: 45px;
+  order: 1;
+  margin-top: 65px;
 }
 
 .logo {
-    align-self: flex-start;
-    font-family: Poppins;
-    height: 25px;
-    margin-left: 20px;
-    font-family: Poppins;
-    font-weight: 700;
-    font-size: 18px;
-    text-align: center;
-    color: #6070FF;
+  align-self: flex-start;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
```

Comment:
- [ ] Kindly use `  font-family: poppins, sans-serif;` only in all the cases below instead of using `font-family: "Poppins", Arial, Helvetica, sans-serif;`
  - In your project `fonts` are not working as it should be. Kindly consider my suggestion.

***Your Desing***
![Screenshot from 2023-01-12 02-08-32](https://user-images.githubusercontent.com/60083437/211907476-466f78b6-50cf-4353-9f2c-948680adfd10.png)

***Figma Design***
![Screenshot from 2023-01-12 02-05-40](https://user-images.githubusercontent.com/60083437/211907177-7db66cdd-6b79-4eb8-8d26-bf462e89e700.png)